src/stylesheets/components/_components.popup.scss

```diff
@@ -0,0 +1,37 @@
+.c-popup {
+  display: none;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 2;
+  background-color: $tundora;
+  &.is-active {
+    display: block;
+  }
+}
+
+.c-popup__wrapper {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: $white;
+  box-shadow: 0 0 4px $ebony-clay;
+  padding: 40px 5% 62px 5%;
```

Comment:
@xavierang not sure about this % setting on the padding probably allows some un-necessary flexibility.